### PR TITLE
(PC-9902): compute subcategory from type in offer serialization

### PR DESCRIPTION
**Pull request by Aliochka**

### CODE REVIEW

File: src/pcapi/core/categories/subcategories.py [hunk 1]
```diff
@@ -1053,6 +1053,76 @@ def __post_init__(self):
...
 )
 # endregion
+# region TECHNICAL
+ACTIVATION_EVENT = Subcategory(
+    id="ACTIVATION_EVENT",
+    category_id=categories.TECHNIQUE.id,
+    matching_type="EventType.ACTIVATION",
+    pro_label="Catégorie technique d'événement d'activation ",
+    app_label="Catégorie technique d'événement d'activation ",
+    search_group=SearchGroupChoices.MATERIEL.value,
```
Comment (francois-seguin) [inline src/pcapi/core/categories/subcategories.py:1063]:
```suggestion
    search_group=SearchGroupChoices.NONE.value,
```

cf https://github.com/pass-culture/pass-culture-api/blob/0caab9622e7d900e0e210fca18316fe4aecb3779/src/pcapi/core/offers/models.py#L485

En ajoutant   `NONE = None` à `SearchGroupChoices`

File: src/pcapi/core/categories/subcategories.py [hunk 2]
```diff
@@ -1053,6 +1053,76 @@ def __post_init__(self):
...
+    reimbursement_rule=ReimbursementRuleChoices.NOT_REIMBURSED.value,
+    is_selectable=False,
+)
+ACTIVATION_THING = Subcategory(
+    id="ACTIVATION_THING",
+    category_id=categories.TECHNIQUE.id,
+    matching_type="ThingType.ACTIVATION",
+    pro_label="Catégorie technique de thing d'activation",
+    app_label="Catégorie technique de thing d'activation",
+    search_group=SearchGroupChoices.MATERIEL.value,
```
Comment (francois-seguin) [inline src/pcapi/core/categories/subcategories.py:1080]: idem plus haut

File: src/pcapi/core/categories/subcategories.py [hunk 3]
```diff
@@ -1053,6 +1053,76 @@ def __post_init__(self):
...
+    reimbursement_rule=ReimbursementRuleChoices.NOT_REIMBURSED.value,
+    is_selectable=False,
+)
+JEU_SUPPORT_PHYSIQUE = Subcategory(
+    id="JEU_SUPPORT_PHYSIQUE",
+    category_id=categories.TECHNIQUE.id,
+    matching_type="ThingType.JEUX",
+    pro_label="Catégorie technique Jeu support physique",
+    app_label="Catégorie technique Jeu support physique",
+    search_group=SearchGroupChoices.MATERIEL.value,
```
Comment (francois-seguin) [inline src/pcapi/core/categories/subcategories.py:1097]:
```suggestion
    search_group=SearchGroupChoices.JEU.value,
```

File: src/pcapi/core/categories/subcategories.py [hunk 4]
```diff
@@ -1053,6 +1053,76 @@ def __post_init__(self):
...
+    reimbursement_rule=ReimbursementRuleChoices.NOT_REIMBURSED.value,
+    is_selectable=False,
+)
+OEUVRE_ART = Subcategory(
+    id="OEUVRE_ART",
+    category_id=categories.TECHNIQUE.id,
+    matching_type="ThingType.OEUVRE_ART",
+    pro_label="Catégorie technique Oeuvre d'art",
+    app_label="Catégorie technique Oeuvre d'art",
+    search_group=SearchGroupChoices.MATERIEL.value,
```
Comment (francois-seguin) [inline src/pcapi/core/categories/subcategories.py:1114]:
```suggestion
    search_group=SearchGroupChoices.NONE.value,
```

File: src/pcapi/core/categories/subcategories.py (hunk 1 above)
Comment (Aliochka) [reply src/pcapi/core/categories/subcategories.py:1063]: :+1:

File: src/pcapi/core/categories/subcategories.py (hunk 2 above)
Comment (Aliochka) [reply src/pcapi/core/categories/subcategories.py:1080]: :+1:

File: src/pcapi/core/categories/subcategories.py (hunk 3 above)
Comment (Aliochka) [reply src/pcapi/core/categories/subcategories.py:1097]: :+1:

File: src/pcapi/core/categories/subcategories.py (hunk 4 above)
Comment (Aliochka) [reply src/pcapi/core/categories/subcategories.py:1114]: :+1:

**Review by francois-seguin — COMMENTED** [review]
Une retouche de sérialisation est nécessaire:
```
class SubcategoryResponseModel(BaseModel):
    ...
    search_group: Optional[str]
```

**Review by francois-seguin — APPROVED** [review]

**Pull request merged**